### ISSUE TITLE
Accessibility Considerations

### ISSUE THREAD

**Issue by AustinPaquette** [paquette/ui#190]
Accessibility options and defaults should be brought into the forefront of how the Paquette UI works.

Examples of best practices (documentation) as well as working with the various HTML attributes -- such as `aria-*` -- should be researched.
